lib/tag-field.jsx

```diff
@@ -131,7 +135,7 @@ export default React.createClass( {
 					onKeyDown={this.onKeyDown}
 					onBlur={this.onBlur}>
 					<input className="hidden-tag" tabIndex="-1" ref="hiddenTag" onKeyDown={this.preventTyping} />
-					{this.props.tags.map( ( tag, index ) =>
+					{ uniq( tags ).map( ( tag, index ) =>
```

Comment:
it was a drive-by fix. there's a bigger issues I've discovered with tags wherein we're getting multiple tags with the same label. you are absolutely right in that we need to fix this at its root, but I threw this in in the meantime to prevent the errors from crashing things